superset/assets/src/visualizations/partition.js

```diff
@@ -221,26 +250,52 @@ function partitionVis(slice, payload) {
                     `background-color: ${n.color};'` +
                 '></div>' +
               '</td>' +
+              `<td>${getCategory(n.depth)}</td>` +
               `<td>${n.name}</td>` +
               `<td>${n.disp}</td>` +
-              `<td>${getCategory(n.depth)}</td>` +
             '</tr>'
           );
         });
+      } else {
+        t += (
+          '<thead><tr><td colspan="3">' +
+            `<strong class='x-value'>${getCategory(d.depth)}</strong>` +
+            '</td></tr></thead><tbody>'
+        );
+        t += (
+          '<tr class="emph">' +
+            '<td class="legend-color-guide" style="opacity: 0.75">' +
```

Comment:
do styles for `.legend-color-guide` and `.x-value` come from another style sheet? or do they do anything?